Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -13,7 +13,8 @@ define([
         '../Core/WebMercatorTilingScheme',
         '../ThirdParty/Uri',
         './GetFeatureInfoFormat',
-        './UrlTemplateImageryProvider'
+        './UrlTemplateImageryProvider',
+        '../Scene/TimeDynamicImagery'
```

Comment:
`TimeDynamicImagery.js` is in the same directory. No need to the `../Scene`

Reply:
thanks, will make this change